apidocs/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../.eslintrc",
+  "rules": {
+    "quotes": ["off"]
```

Comment:
How about re-enabling this check via inline directive inside describe-builtin-models.js? Something like this (put it at the top):

```js
/* eslint-enable quotes */
```

Not a big deal though.